source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
+
+## What are our Core Values?
+
+- **Collaboration:** Foster a culture of open communication and collective problem-solving.
+- **Quality:** Uphold high standards in our work, ensuring reliability and efficiency.
+- **Innovation:** Embrace new ideas and technologies that help us improve our service offering.
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress.
+
+## Team Roles
+
+- **Team Members:** Comprising a mix of developers, data and platform engineers,
+responsible for the delivery of tasks and active participation in all Agile ceremonies.
+- **Product Manager (PM):** Oversees the product backlog,
+ensuring it aligns with stakeholder needs and business objectives.
+- **Delivery Manager (DM):** Facilitates the Agile process,
+removing blockers and ensuring timelines are met.
+
+## Communication and Delivery
+
+- **Daily [Stand-ups](#stand-up):** A quick get-together to share progress updates.
+- **Ceremonies:** How we make sure our team stays aligned as we work to deliver during Agile Sprints.
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Fortnightly Sprints** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence   | Day             | Time          |
+| --------------------------------------------- | ----------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily       | Monday - Friday | 10:45 - 11:00 |
+| [Show & Tell](#show-and-tell)                 | Fortnightly | Monday          | 15:00 - 15:30 |
+| [Roadmap Monthly Review](#roadmap-review)     | Fortnightly | Monday          | 15:30 - 16:15 |
+| [Refinement 1](#refinement)                   | Fortnightly | Monday          | 13:00 - 14:00 |
+| [Peer Review](#peer-review)                   | Fortnightly | Thursday        | 15:30 - 16:30 |
+| [Refinement 2](#refinement)                   | Fortnightly | Thursday        | 12:00 - 12:45 |
+| [Sprint Retrospective](#sprint-retrospective) | Fortnightly | Thursday        | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Fortnightly | Thursday        | 13:00 - 14:00 |
+
```

Comment:
Is adding the ceremony meeting schedule creating a bit of a doc maintenance headache?

Reply:
Potentially, yes, but also I think it's good to actually see it and state what it is. I didn't realise the precise order of operations until I put it all down here. Might be worth the headache.